libraries/AP_Common/tests/test_bitmask.cpp

```diff
@@ -36,6 +42,16 @@ TEST(Bitmask, Tests)
     EXPECT_EQ(48, x.first_set());
     x.clear(48);
     EXPECT_EQ(-1, x.first_set());
+
+    x.set(50);
+    EXPECT_EQ(hal.util->persistent_data.internal_errors, static_cast<uint32_t>(AP_InternalError::error_t::bitmask_range));
+    EXPECT_EQ(hal.util->persistent_data.internal_error_count, 1);
+    EXPECT_NE(hal.util->persistent_data.internal_error_last_line, 0);
```

Comment:
These are out-of-place for a unit test named "test_bitmask".

Reply:
I don't agree much as this is what we set on failure so we should be testing that are setting the right things on failure. But anyways, I have remove it

Reply:
> I don't agree much as this is what we set on failure so we should be testing that are setting the right things on failure. But anyways, I have remove it

That's all good - but doesn't belong in `test_bitmask` - a `test hal`, perhaps :-)